### PR TITLE
Added environment indicator

### PR DESCRIPTION
**Pull request by svdgraaf**
Added an indicator icon, so you can scan and see which accounts are sensitive...

screenshot 2017-08-01 16 40 06

### CODE REVIEW

**Comment by svdgraaf** [comment]
^^ travis setup is borked

**Comment by dennisvink** [comment]
I'm not sure if I like the aesthetics of the exclamation marks. I would prefer to colourise the already existing circled 'I' element.

File: app/partials/bookmarks/list.html [hunk 1]
```diff
@@ -19,7 +19,10 @@
 	>
 		<img ng-if="!gravatar(bookmark)" ng-src="{{bookmark.avatar_url}}" />
 		<img ng-if="gravatar(bookmark)" gravatar-src="bookmark.avatar_url || bookmark.name" />
-		<h2>{{bookmark.name}}</h2>
+		<h2>
+			<span class="icon ion-alert red" ng-if="is_environment(bookmark, 'production')" />
+			<span class="icon ion-alert orange" ng-if="is_environment(bookmark, 'acceptance')" />
```
Comment (dennisvink) [inline app/partials/bookmarks/list.html]: The colours and keywords should be configurable on the settings page, imho.

**Comment by svdgraaf** [comment]
@dennisvink Hmmm, I was actually looking for a different icon altogether, but the used icon library is a bit lacking: http://ionicons.com/

I'll see if can make things a bit more dynamic

**Comment by svdgraaf** [comment]
It now switches the info icon to a different color

screenshot 2017-08-02 15 31 39

**Comment by dennisvink** [comment]
Looks good!

**Comment by svdgraaf** [comment]
As requested, now replaces the role name with the environment name.
screenshot 2017-08-21 16 40 00

**Comment by tvb** [comment]
👍 LGTM

**Pull request merged**